app/models/badges/v1_badge.rb

```diff
@@ -0,0 +1,21 @@
+module Badges
+  class V1Badge < Badge
+    seed "V1",
+      :common,
+      'v1',
+      'Awarded for having joined Exercism v1'
```

Comment:
```suggestion
      'Awarded for having joined when Exercism was at version 1'
```

(And the same for v2)